src/main/arm/mainTemplate.json

```diff
@@ -329,7 +329,7 @@
                     "adminUsername": "[parameters('adminUsername')]",
                     "adminPassword": "[parameters('adminPasswordOrKey')]",
                     "linuxConfiguration": "[if(equals(parameters('authenticationType'), 'password'), json('null'), variables('const_linuxConfiguration'))]",
-                    "customData": "[base64(concat(parameters('ibmUserId'), ',', parameters('ibmUserPwd')))]"
+                    "customData": "[base64(concat(parameters('ibmUserId'), ' ', parameters('ibmUserPwd')))]"
```

Comment:
Can we only use one `customData` parameter?

Reply:
@git4rk I am not sure I 100% understand your question, but we must use exactly **one** `customData` parameter because this is required by **cloud-init**.